src/app/content/highlights/components/Highlights.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import myHighlightsEmptyImage from '../../../../assets/MHpage-empty-logged-in.png';
+import Loader from '../../../components/Loader';
+import { LinkedArchiveTreeNode } from '../../types';
+import { archiveTreeSectionIsChapter, findArchiveTreeNode } from '../../utils/archiveTreeUtils';
+import { stripIdVersion } from '../../utils/idUtils';
+import { highlightLocations, summaryFilters, summaryHighlights, summaryIsLoading } from '../selectors';
+import { SummaryHighlights } from '../types';
+import * as HStyled from './HighlightStyles';
+import * as Styled from './ShowMyHighlightsStyles';
+
+// tslint:disable-next-line: variable-name
+const Highlights = () => {
+  const filters = useSelector(summaryFilters);
+  const locations = useSelector(highlightLocations);
+  const highlights = useSelector(summaryHighlights);
+  const isLoading = useSelector(summaryIsLoading);
+
+  if (
+    !isLoading &&
+    (filters.colors.length === 0 || filters.locationIds.length === 0)
+  ) {
+    return <Styled.Highlights>
+      <HStyled.GeneralCenterText>
+        <FormattedMessage id='i18n:toolbar:highlights:popup:heading:no-highlights'>
+          {(msg: Element | string) => msg}
+        </FormattedMessage>
+        <span>
+          <FormattedMessage
+            id='i18n:toolbar:highlights:popup:heading:no-highlights-tip'
+            defaultMessage='Try selecting different chapter or color filters to see different results.'
```

Comment:
Not sure how this library works, but when I was using [Fluent](https://projectfluent.org/) I had to specify message because when provided `id` was missing it would break app :D

Reply:
there will be an error if the message isn't found in the configuration, but that just means we need to add the message to the configuration